schema/me/conversation.js

```diff
@@ -173,12 +202,24 @@ export const ConversationFields = {
     type: connectionDefinitions({ nodeType: MessageType }).connectionType,
     description: "A connection for all messages in a single conversation",
     args: pageable(),
-    resolve: ({ messages }, options) => {
-      const impulseOptions = parseRelayOptions(options)
-      return connectionFromArraySlice(messages, options, {
-        arrayLength: messages.length,
-        sliceStart: impulseOptions.offset,
-      })
+    resolve: ({ id, from_email_address }, options, req, { rootValue: { accessToken } }) => {
+      const { page, size, offset } = parseRelayOptions(options)
+      const impulseParams = { page, size, conversation_id: id }
```

Comment:
We call Impulse with the `conversation_id`, plus pagination params. We use `parseRelayOptions` to take Relay cursor-style pagination and transform it to `page`, `size` and `offset`.

The API takes combinations of `page` and `size`, but we need `offset` a few lines below for the `sliceStart` Relay helper...../shrug